enhancements/baremetal/enable-baremetal-on-azure-gcp.md

```diff
@@ -0,0 +1,159 @@
+
+2. documentation will need to be added to the centralized host management documentation explaining how to create and update a Provisioning CR for the Azure and GCP platforms.
+
+### Workflow Description
+
+#### Variation
+
+### User Stories
+
+#### Story 1 - Current IPI baremetal platform use case
+
+No change.
+
+#### Story 2 - centralized host management use case
+
+As a user of a hub cluster that performs central infrastructure management, and optionally zero-touch provisioning, I need to provision hosts using the k8s-native API (Baremetal Hosts CR) even when the hub cluster has a platform of Azure or GCP.
+
+### Risks and Mitigations
+
+One concern of this implementation is that for the solution to work properly the inbound and outbound connections of the BMCs of the bare metal hosts deployed and managed in this way will be exposed externally, as they need to communicate with the control plane, hosted in the Azure or GCP platforms.
+It is essential to use some vpn/tunnel-like connection between the bare metal hosts and the control plane.  
+For example, a native solution commonly used in Azure is the [Azure Virtual Network (VNet)](https://learn.microsoft.com/en-us/azure/virtual-network/virtual-networks-overview), while on Google Cloud Platform is the [GCP VPC](https://cloud.google.com/vpc).  
+A risk is given by the fact that we don't have a way to verify the correctness of this configuration when directly managed by the user.  
+A way to mitigate this is to provide accurate documentation and guidelines.
+Since the only deployment method supported by this feature is using virtual media, disabling IPMI on the hosts BMCs is also highly recommended since it's of no use.
+
+### Drawbacks
+
+## Design Details
+
+### API Extensions
+
+### Operational Aspects of API Extensions
+
+#### Failure Modes
+
+* Misconfiguring the Provisoning CR will end up in failure during bare metal hosts deployment. See the [Proposal section](#Proposal) on how to correctly configure the Provisoning CR, especially the provisioningNetwork parameter.
+* If the BMC access is misconfigured, the enrollment of the associated bare metal hosts will not happen, or there will be failures during the inspection phase.
+
+#### Support Procedures
+
+### Test Plan
+
+#### Unit Testing
+
+We will add unit tests to confirm that cluster-baremetal-operator:
+* is enabled on the Azure and GCP platforms
+* will restrict functionality on the Azure and GCP platforms to ProvisioningNetork=Disabled
+
+#### Functional Testing
+
+QE will validate that the Azure and GCP platforms are supported to reduce the load on CI.
+
+### Graduation Criteria
+
+#### Dev Preview -> Tech Preview
+
+#### Tech Preview -> GA
+
+The feature was silently enabled in Openshift 4.14 and it will go to GA in Openshift 4.14.  
```

Comment:
Copy-paste from AWS, does not make sense here.

Reply:
whoops forgot to change that

Reply:
I fixed the versions, should be ok now